apps/passport-server/src/services/telegramService.ts

```diff
@@ -213,51 +173,9 @@ export class TelegramService {
           const firstName = ctx?.from?.first_name;
           const name = firstName || username;
           await ctx.reply(
-            `Welcome ${name}! 👋\n\nClick below to ZK prove that you have a ticket to an event, so I can add you to the attendee Telegram group!\n\nYou must have one of the following tickets in your Zupass account to join successfully.\n\nSee you soon 😽`
-          );
-          const msg = await ctx.reply(`Loading tickets and events..`);
-          const events = await fetchLinkedPretixAndTelegramEvents(
-            this.context.dbPool
-          );
-          const eventsWithChatsRequests = events.map(async (e) => {
-            return {
-              ...e,
-              chat: e.telegramChatID
-                ? await this.bot.api.getChat(e.telegramChatID)
-                : null
-            };
-          });
-          const eventsWithChatsSettled = await Promise.allSettled(
-            eventsWithChatsRequests
+            `Welcome ${name}! 👋\n\nClick below join a TG group via a ZK proof!\n\nYou will sign in to Zupass, then prove you have a ticket for one of the events associated with the group.\n\nSee you soon 😽`,
```

Comment:
revised copy for choosing a group to join instead of seeing which events you connect to which groups